website/utilitypage_mine.py

```diff
@@ -52,8 +54,11 @@ def search():
             latitude, longitude = geocode(addressfull)
             utility = correct_utility_function(latitude, longitude)
             (utility3, link2) = utility
-            mockGageData = pd.read_csv(r"/Users/benwilliams/Documents/Data+/nwis_data_mock.csv", header=0, names=['date', 'value', 'data_type', 'nwis_site_no'])
-        numberToName = pd.read_csv(r"/Users/benwilliams/Documents/Data+/nwis_site_info.csv")
+            # mockGageData = pd.read_csv(r"/Users/benwilliams/Documents/Data+/nwis_data_mock.csv", header=0, names=['date', 'value', 'data_type', 'nwis_site_no'])
+            folder = 'C:\\Users\\wyseg\\'
```

Comment:
Can we fix this so that it reads from a relative path? Preferably a local copy. If you check the files into git, please use git lfs so we don't have huge files.

Reply:
Can I just throw that all away? That's from back when we weren't using the SQL database

Reply:
Oh riiiight. Yeah, do that.